Mathlib/Order/RelSeries.lean

```diff
@@ -651,6 +651,18 @@ noncomputable def comap (p : LTSeries β) (f : α → β)
   LTSeries α := mk p.length (fun i ↦ (surjective (p i)).choose)
     (fun i j h ↦ comap (by simpa only [(surjective _).choose_spec] using p.strictMono h))
 
+/-- The strict series `0 < … < n` in ℕ. -/
+def iota (n : ℕ) : LTSeries ℕ where
```

Comment:
It's inspired by `List.iota`; that's the only reason.

Reply:
@nomeata `iota`  actually produces numbers in _decreasing_ order. You are thinking about `List.range`.

It would be nice to generalise what should be called `LTSeries.range` to other starts, ends and even steps, based on either `List.range` or `Finset.Icc`.

Reply:
Oh, it does? Sorry for that, will fix.

As for the generalization - in my application I use `.map` to shift it around. But you are right, the generalization seem to add reasonable convenience.

Reply:
Hmm,  not sure if `range (start := 0 : Nat) (end : Nat)` is nice; the length would be Nat subtraction, and having that in the types (`Fin (end-start+1)`) is probably not nice.

But this might be ok
```lean
/-- The strict series `start < … < start+n` in `ℕ`. -/
def range (n : ℕ) (start := 0) : LTSeries ℕ where
  length := n
  toFun := fun i => start + i
  step i := Nat.lt_add_one _
```
updated the PR to this.

Reply:
I don't really care about generalising `LTSeries.range` to nonzero start points either.